extensions/1loc/src/index.tsx

```diff
@@ -22,15 +23,17 @@ const useCategories = (): [ICategory[], boolean, AxiosResponse | null] => {
     const fetchCategories = async () => {
       cancelRef.current?.cancel()
       cancelRef.current = axios.CancelToken.source()
-      const url = "https://api.github.com/repositories/251039251/contents/snippets"
+      const url = "https://api.github.com/repositories/251039251/contents/contents"
       const config = {
         cancelToken: cancelRef.current?.token,
         headers: personalAccessToken ? { Authorization: `token ${personalAccessToken}` } : undefined,
       }
       try {
-        const { data } = await axios.get(url, config)
+        const { data } = await axios.get<ICategory[]>(url, config)
         if (isMounted.current) {
-          setCategories(data)
+          const filtered = data.filter((item) => item.type === "dir" && !item.name.startsWith("_"))
```

Comment:
The extension considers folders as categories, thus we need to filter out files. Additionally, folders named `_<name>` do not contain any snippets and should also be filtered out.

See their [project structure](https://github.com/phuocng/1loc/tree/master/contents) for reference.